src/client/js/data/fetching.test.js

```diff
@@ -0,0 +1,52 @@
+const {fetchData} = require('./fetching')
+
+describe('fetchData', () => {
+  let mockedSuccess = true
+  let mockedPayload = {}
+  let mockedResponse = {
+    ok: mockedSuccess,
+    ...mockedPayload,
+    json: function() {
+      return mockedPayload
+    }
+  }
+
+  beforeEach(() => {
+    global.fetch = jest.fn().mockImplementation(() => {
```

Comment:
Is there a preferred fetch mocking method we can use?

Reply:
yeah, this is the only use of fetch on the client, so not sure if it is worth it